pkg/controller/kubedirectorcluster/members.go

```diff
@@ -1081,12 +1086,17 @@ func queueNotify(
 		op = "addnodes"
 		deltaFqdns = fqdnsList(cr, creatingOrCreated)
 	}
+
 	if op == "" {
 		if deletePending, ok := modifiedRole.membersByState[memberDeletePending]; ok {
 			op = "delnodes"
 			deltaFqdns = fqdnsList(cr, deletePending)
+		} else {
+			stateDetail.LastSetupGeneration = stateDetail.LastConfigDataGeneration
+			return
```

Comment:
I don't think we need/want to change queueNotify anymore, now that syncMemberNotifies has been changed.